DLA_finder/data_model/Dla.py

```diff
@@ -0,0 +1,8 @@
+""" Simple object to hold DLAs"""
```

Comment:
Isn't this code *identical* to that inside `dla_cnn/data_model` ?
I don't understand what we are doing.